File: bindings/go/binding_test.go
package tree_sitter_osgi_bundle_manifest_test

import (
	"testing"

	tree_sitter "github.com/tree-sitter/go-tree-sitter"
	tree_sitter_osgi_bundle_manifest "github.com/impure0xntk/tree-sitter-osgi-bundle-manifest/bindings/go"
)

func TestCanLoadGrammar(t *testing.T) {
	language := tree_sitter.NewLanguage(tree_sitter_osgi_bundle_manifest.Language())
	if language == nil {
		t.Errorf("Error loading OsgiBundleManifest grammar")
	}
}

File: bindings/node/binding_test.js
const assert = require("node:assert");
const { test } = require("node:test");
const Parser = require("tree-sitter");

test("can load grammar", () => {
  const parser = new Parser();
  assert.doesNotReject(() => {
    const language = require("./index.js").default;
    parser.setLanguage(language);
  });
});
